Assignment/Q1_Assignment_1.txt

```diff
@@ -0,0 +1,21 @@
+let obj = {
+ name:"Harry Potter",
+ age: 12,
+ address: {
+  details: ["4","Privet Drive"],
+  area:"Little Whinging",
+  city: "Surrey",
+  state: "England"
+ } 
+}
+let clone = structuredClone(Object.assign({},obj,{address:{details:["5","Privet Drive"]}}));
```

Comment:
As you are updating the address object with new object inside Object.assign you don't need to use structuredClone